core/src/win32/filed/vss_generic.cc

```diff
@@ -85,7 +85,7 @@ class IXMLDOMDocument;
 #  include "Win2003/vswriter.h"
 #  include "Win2003/vsbackup.h"
 
-#  define VSS_ERROR_OBJECT_ALREADY_EXISTS 0x8004230D
+#  define VSS_ERROR_OBJECT_ALREADY_EXISTS ((HRESULT)0x8004230D)
```

Comment:
I wonder where this was taken from.  Its not listed as an official return value:
* https://learn.microsoft.com/de-de/windows/win32/api/vsbackup/nf-vsbackup-ivssbackupcomponents-addtosnapshotset

Maybe this was just trial & error ?
Its probably there for a good reason though, so we should not remove it.